watchfaces/000-default-digital.qml

```diff
@@ -27,6 +28,16 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+/*
+ * Based on 000-default-digital, changes:
+ * Center hour horizontally and vertically
+ * Minutes are now ~half the size of hours and centered to top half of hourCanvas
+ * AM/PM moved up/right to compesate spacing to bigger minutes and horiz. centered to top of hour
+ * date is now horiz. centered to bottem 5/6 of hour
+ * sliced AM/PM to 2 chars to prevent localized version with more chars
+ * feed AP text only with "en_EN"
+ */
```

Comment:
You shouldn't justify your changes in the source code itself. Git is there for that, if you create atomic commits with a good commit message, git log should be where we find that kind of information.
Plus, this is not "based on 000-default-digital" anymore, this *is* 000-default-digital.
This comment is also valid for a couple of other watchfaces in this PR

Reply:
maan you are fast :)

removing comments. Mostly left-overs from my learning.

The svg is unoptimized, correct. changed some colors and forgot to reoptimize this one.

Regarding atomic commits with good messages. right, i should do but my "workflow" is quite amateurish and i am happy to get any results at all up to this point